packages/vuetify/src/components/VList/VListSubheader.tsx

```diff
@@ -13,30 +13,36 @@ export const VListSubheader = defineComponent({
     color: String,
     inset: Boolean,
     sticky: Boolean,
-    title: String,
+    text: String,
```

Comment:
Why not title to keep it consistent with v-list-item?

Reply:
Because it's a subheading. By definition it wouldn't be a title.

Reply:
> I think **list** could simply be a global composable, or even **depth**. Could use it to control the stacking order for various floating elements. Everything else looks good.

No need to make those changes now. Let's see if the need arises in the future

Reply:
It's the title of the subheading? 🤷